Day 2 - Python Datetime objects and Timedelta/Q1_BeforeChristmas.py

```diff
@@ -1,10 +1,10 @@
+
+# Problem to solve!
 """
 1. What date is 3 weeks and 4 days before Christmas?
 Solution provided by Sanjay Siddha
 """
 
```

Comment:
Don't add any changes here. revert these files to the way they were. Use `git pull --rebase origin master`.